content/developers/api-reference/events-api/index.md

```diff
@@ -391,6 +391,90 @@ You should see the response:
 }
 ```
 
+### Event Record Retrieval
+
+Event records in DataTrails are tokenized at creation time and referred to in all API calls and smart contracts throughout the system by a unique identity of the form:
+
+```bash
+assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+```
+
+If you do not know the Event’s identity you can fetch Event records using other information you do know.
+
+#### Fetch All Events
+
+To fetch all Event records, simply `GET` the Events resources:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivist/v2/assets/-/events"
+```
+
+#### Fetch Events for a Specific Asset
+
+If you know the unique identity of the Asset record simply `GET` the resource:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivist/v2/assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events"
+```
+
+#### Fetch Specific Events by Identity
+
+If you know the unique identity of the Asset and Event record simply `GET` the resource:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivst/v2/assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+```
+
+#### Fetch Event by Type
+
+To fech all Events of a specific type, `GET` the Events resource and filter on `arc_display_type`:
+
+```bash
+curl -g -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivist/v2/assets/-/events?event_attributes.arc_display_type=Software%20Package%20Release"
+```
+
+#### Fetch Event by Asset Type
```

Comment:
There seem to be 2 concepts here - the test is talking about fetch by Asset type (like, show me all events that have happened to "apples") but the example then shows fetching based on an Asset _property_ (like, show me all events that have happened where the Asset has been "peeled"). 

Would be good to split these out and have both - so an example o go with what the text ays (arc_display_type) and some text to accompany the example (asset_attributes.foo)

Reply:
Hi @JAG-UK   . . . there already is an example that fetches an Event by Asset Type it's after Fetch Event by type.  However I do see an update to the verbiage. . . but it does already exists. . . provided that we are talking about the same two examples.

Reply:
Hello @JAG-UK  . . . ok hopefully we are referencing the same thing re: the above about updating the verbiage as there has been additional commits to change from asset type to asset attribute. . . did that last week.  Let me know if this is what you are referring to.